Sprint-3/reading-list/script.js

```diff
@@ -20,4 +20,47 @@ const books = [
     bookCoverImage: "https://blackwells.co.uk/jacket/l/9780135957059.jpg",
   },
 ];
+const firstBook = document.getElementById("firstBookDiv");
+const secondBook = document.getElementById("secondBookDiv");
```

Comment:
this is good, but the only issue here is that this would work only with 3 books. What if new books were added in the future? you would have to add a lot of extra code. Instead, you can just create a list item for every book!